lms/templates/themable-footer.html

```diff
@@ -0,0 +1,14 @@
+## mako
```

Comment:
There have been lots of questions about the structure of these header/footer/themable whatevers.  Would comments at the top help people understand them?